gloo/docs/gloo_routing/virtual_services/authentication/external_auth/test-no-auth-vs.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: test-no-auth
   namespace: gloo-system
 spec:
+  displayName: test-no-auth
```

Comment:
i noticed this was missing after a dry-run of the `glooctl` command, so I added it. but what is this used for? in what way is the actual name insufficient, and we want a separate display name?

Reply:
it's just used by the UI to show a more friendly name; I don't include it in the examples because it seems unnecessary to demonstrate the functionality